BACKWARDS_COMPATABILITY.md

```diff
@@ -0,0 +1,116 @@
+     - [Capabilities](https://docs.pepr.dev/main/user-guide/capabilities/)
+     - [Store](https://docs.pepr.dev/main/user-guide/store/)
+     - [OnSchedule](https://docs.pepr.dev/main/user-guide/onschedule/)
+1. **Configuration**:
+   - YAML or JSON configuration structures for [Pepr Modules](https://docs.pepr.dev/main/user-guide/pepr-modules/). Configuration includes:
+     - Environment Variables
+       - [Redact Store Values from Logs](https://docs.pepr.dev/main/user-guide/customization/#redact-store-values-from-logs)
+       - [Display Node Warnings](https://docs.pepr.dev/main/user-guide/customization/#display-node-warnings)
+       - [Customizing Log Format](https://docs.pepr.dev/main/user-guide/customization/#customizing-log-format)
+       - [Customizing Watch Configuration](https://docs.pepr.dev/main/user-guide/customization/#customizing-watch-configuration)
+       - [Configuring Reconcile](https://docs.pepr.dev/main/user-guide/customization/#configuring-reconcile)
+       - [Customizing with Helm](https://docs.pepr.dev/main/user-guide/customization/#customizing-with-helm)
+       - [Customizing with package.json](https://docs.pepr.dev/main/user-guide/customization/#customizing-with-packagejson)
+1. **Pepr SDK**:
+   - Helper functions to frequently used tasks defined in [SDK](https://docs.pepr.dev/main/user-guide/sdk/).
+1. **CLI Commands**:
+   - Commands, flags, and outputs provided by the [Pepr CLI](https://docs.pepr.dev/main/user-guide/pepr-cli/).
+
+---
+
+## Backward Compatibility Guarantees
+
+Pepr adheres to the following guarantees to ensure a seamless user experience:
+
+### 1. Semantic Versioning (SemVer)
+Pepr follows SemVer conventions:
+- **Major Versions (X.0.0)**:
+  - Not currently in use until we get to 1.0.
+- **Minor Versions (X.Y.0)**:
+  - Add new features in a backward-compatible manner.
+  - Introduce breaking changes after deprecation.
+- **Patch Versions (X.Y.Z)**:
+  - Introduce deprecations changes with announcements and migration paths.
+  - Include bug fixes, security updates, and performance improvements without changing functionality or APIs.
+
+### 2. Deprecation Policy
+- **Deprecation Announcement**:
+  - Any feature slated for removal will be marked as deprecated and announced in release notes.
+  - Deprecated features will include clear warnings in logs and CLI outputs.
+  - Deprecation notices will be included [Pepr Slack](https://kubernetes.slack.com/archives/c06dgh40ucb).
+- **Grace Period**:
+  - Deprecated features will remain functional for at least **one release** before removal.
+
+### 3. Test Coverage
+- Maintain backward compatibility by validating changes against existing test suites for previously supported versions.
+- Include regression tests to ensure deprecated features continue to function as expected until their removal.
+
+---
+
+## Handling Breaking Changes
+
+### 1. Communication
+- Breaking changes will only be introduced in minor releases.
```

Comment:
```suggestion
- Breaking changes will only be introduced in major releases.
```

Pretty sure you mean Major, per semver:
> MAJOR version when you make incompatible API changes
> MINOR version when you add functionality in a backward compatible manner

Reply:
While that is correct, we do not follow the actual semver standards until we hit v1 so we actually do breaking changes if there are in minors since we can't touch major. I should add a note that pertaining to that around the `Pepr follows SemVer conventions:`

Reply:
Yeah, a clarification is useful here. At first glance it sounds like a direct conflict with what semver requires.